### ISSUE TITLE
NPE - android.widget.EditText in databinding.ActivityContinuousScanBinding.quickViewSearchByBarcode' views.ContinuousScanActivity.hideAllViews (line 490)

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#3246]
Attempt to read from field 'android.widget.EditText *.openfood.databinding.ActivityContinuousScanBinding.quickViewSearchByBarcode' on a null object reference at *.openfood.views.ContinuousScanActivity.hideAllViews(ContinuousScanActivity.java:490)

app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ContinuousScanActivity.java

```
2020-06-12 17:03:56.116 17149-17149/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: openfoodfacts.github.scrachx.openbeauty, PID: 17149
    java.lang.NullPointerException: Attempt to read from field 'android.widget.EditText openfoodfacts.github.scrachx.openfood.databinding.ActivityContinuousScanBinding.quickViewSearchByBarcode' on a null object reference
        at openfoodfacts.github.scrachx.openfood.views.ContinuousScanActivity.hideAllViews(ContinuousScanActivity.java:490)
        at openfoodfacts.github.scrachx.openfood.views.ContinuousScanActivity.lambda$onCreate$7$ContinuousScanActivity(ContinuousScanActivity.java:606)
        at openfoodfacts.github.scrachx.openfood.views.-$$Lambda$ContinuousScanActivity$kNT4tS01CpARjbn5Wc_WCURtDk4.run(Unknown Source:2)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7523)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:941)
```

**Comment by VaiTon** [comment]
How did you get this? It means that the binding got invalidated. Could you provide the steps to get this?

**Comment by teolemon** [comment]
I'm not quite sure, I remember swiping back from the Scan view.

**Comment by teolemon** [comment]
[logcat-prelaunch.txt](https://github.com/openfoodfacts/openfoodfacts-androidapp/files/4774733/logcat-prelaunch.txt)

**Comment by teolemon** [comment]
Here is the video:
https://storage.googleapis.com/play-mtaas-results-prod/org.openfoodfacts.scanner/332/blueline-28-fr-portrait/video.mp4?GoogleAccessId=498997737111-pmn4sf5896c460sheh5fv3k1j1cq46t5@developer.gserviceaccount.com&Expires=1592058761&Signature=0/osDRRmb%2BRhzJ4GP7JU3hGfXKHJ3H1KFKg2SOFCoGFIx2E%2BNr1Do%2BNxIOPzgh%2BC84mNptuc9xjOI5It0LHwVg3HP5OuIe1/2QfkN0VqqZiFSqmcT1M3fFEROSYSW9jHxko8%2Bz7aBXy%2BYFFCNBb%2BYsg0lyclX1jM4HBLGwgO87wvsEfvPuEWqZyDCPVXJmu17%2Bzs83A9uodYbW8hzUe%2BKTQm4v4nsfYspN276AEIbdscerJn1yDvuVlFW8x0RceVqvz4neZBxgsM1w37As9J9nBHZGVzod1LtYJpk3OONSkPr6JdE18r0LX7PFn84%2BNObp7aP281Xi1Ti5Y8/ReEIQ%3D%3D

**Comment by teolemon** [comment]
https://github.com/openfoodfacts/openfoodfacts-androidapp/blob/master/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ContinuousScanActivity.java

**Comment by VaiTon** [comment]
Fixed by  091e745
